src/corner-kick/README.md

```diff
@@ -0,0 +1,56 @@
+# Corner Kick
```

Comment:
I like the readme, especially where it gives an overview of the layout. Would it also be possible to list any top-level files that are autogenerated? (Are any of the files committed here autogenerated?). Might be useful just so people can quickly get an idea of what we don't need to worry about. The small json files like `tsconfig.prod.json` and `tsconfig.test.json` might be a bit confusing/misleading otherwise (although I realise you could likely also find this out by googling/learning react).

Reply:
My only comment here is that the README is pretty loosely coupled to the code, and so is likely to get out of date pretty quickly. Accordingly, I think it should just include top level details of general structure/design, basically stuff that's not going to change to much.

Reply:
I agree with Mathew. A number of auto-generated files and build files are unlikely to be modified. It would be beneficial to put them in the README.